nativelib/src/main/resources/scala-native/time_millis.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <sys/time.h>
+
+long long scalanative_current_time_millis() {
+    long long current_time_millis;
+
+#define MILLIS_PER_SEC 1000LL
+#define MICROS_PER_MILLI 1000LL
+
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
```

Comment:
I created issue #1948 to report that the error code for gettimeofday() is never checked.
I am not suggesting a change for this PR, just wanted to note that it had been detected.

Reply:
Good idea. An error in C is like an error of death.